test/extensions/filters/http/lua/lua_filter_test.cc

```diff
@@ -1284,12 +1284,9 @@ TEST_F(LuaHttpFilterTest, HttpCallNoBody) {
   callbacks->onSuccess(request, std::move(response_message));
   EXPECT_EQ(0, stats_store_.counter("test.lua.errors").value());
 }
-
+/*
```

Comment:
Need some help in fixing this test case.

Reply:
Maybe we just need to update the `encodeHeaders_` mock to `sendLocalReply` mock just you did in the `ImmediateResponse` case.